docs/install/index.md

```diff
@@ -0,0 +1,55 @@
+# Installation
+
+To get started with InvenioRDM, you need to install `invenio-cli`, our
+command line tool for creating and updating your instance.
+
+## Pre-Requirements
+
+Some system requirements are needed beforehand:
+
+- [Python](https://www.python.org/) (3.6 only)
```

Comment:
@zzacharo Was it 3.6.2 minimum?

Reply:
Won't wait on this to merge, but specify if more precise version is needed.